about.html

```diff
@@ -0,0 +1,143 @@
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+
+    <div class="header-main">
+      <div class="wrapper">
+        <nav class="main-nav">
+          <div class="icon-nav">
+            <a href="./index.html"><img src="./Images/Nav/picha-removebg-preview.png" alt="logo image" /></a>
+          </div>
+          <div class="icon-menu-mobile">
+            <img src="./Images/Nav/hamburger-icon.svg" alt="Hamburger icon" />
+          </div>
+          <ul class="bottom-nav">
+            <li><a class="active" href="index.html">Home</a></li>
+            <li><a href="#testimonials">Preview</a></li>
+            <li><a href="#partners">Contact</a></li>
+            <li><a href="#">Blog</a></li>
+            <li class="folow"><a href="#">Follow us</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+
+    <div class="nav-mobile">
+      <div class="cross">
+        <img src="./Images/Nav/cross.svg" alt="cross" />
+      </div>
+      <ul class="container">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#info">About</a></li>
+        <li><a href="#events">Events</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <main>
+    <section id="info" class="info">
+      <div class="data">
+        <div class="para">
+          <h1 class="jeblow">A studio that provides amazing photography</h1>
+          <div class="description">
+            <p>
+              Adam Centre Studio was started by Abdimulhin Adan. <br> Main Ofices are at Nairobi, Kenya.
+              <br> Tel: +254759935007 <br>
+              email: adamabdimulhi.001@gmail.com <br>
+              Lunga-lunga Road
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="logo-details">
+        <h1>Adam Centre Studio</h1>
+        <hr>
+      </div>
+      <div class="footer-logo">
+        <img src="./Images/Nav/picha-removebg-preview.png" alt="logo">
+      </div>
+
+    </section>
+
+    <section id="#events" class="events">
+      <div class="past">
+        <h3>See Our past global events</h3>
+        <hr>
+        <div class="event-para">
+          <p>Take a look at our past launch events that took place in KICC, Nairobi and Mombasa.</p>
+        </div>
+      </div>
+
+      <div class="event-photos">
+        <div class="photo1">
+          <img src="./Images/Nav/event1.avif" alt="event1">
+          <div class="photo-text">
+            <h5>2020</h5>
+            <p>Launched our products at KICC, Nairobi</p>
+          </div>
+        </div>
+
+        <div class="photo2">
+          <img src="./Images/Nav/event2.avif" alt="">
+          <div class="photo-text">
+            <H5>2022</H5>
+            <p>Giving credit to our customers at Mombasa</p>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly fix the overflow on this section as it is causing the mobile app view to be distorted
- [ ] Kindly add some margin and padding so as to match the design

Your design
![image](https://user-images.githubusercontent.com/81932478/198341062-11541a81-5eec-4b4d-b05b-500da44a86d3.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/198341218-36c259e7-1736-4ad0-9c0d-33d2dc557806.png)